### ISSUE TITLE
Auto lauch the deamon without login in the VT-2 terminal

### ISSUE THREAD

**Issue by antoine2711** [chromebrew/crew-sudo#4]
Hi,

   I have a question. Is it possible to have the deamon autolaunch without login in the VT-2 Terminal?

Regards, Antoine

**Comment by supechicken** [comment]
Yes, you can achieve it by copying [`autostart/crew-sudo.conf` in this repository](https://github.com/chromebrew/crew-sudo/blob/main/autostart/crew-sudo.conf) to `/etc/init/`, but the rootFS verification needs to be disabled first.

**Comment by gyilrosales** [comment]
> Yes, you can achieve it by copying [`autostart/crew-sudo.conf` in this repository](https://github.com/chromebrew/crew-sudo/blob/main/autostart/crew-sudo.conf) to `/etc/init/`, but the rootFS verification needs to be disabled first.

Hello! I'm currently confused on how to run the daemon of crew-sudo and then proceed from there, could you clarify the next steps? (Forgive me, I'm not the brightest when it comes to such.)

**Comment by supechicken** [comment]
> Hello! I'm currently confused on how to run the daemon of crew-sudo and then proceed from there, could you clarify the next steps? (Forgive me, I'm not the brightest when it comes to such.)

Sorry for the late reply! Copy the `autostart/crew-sudo.conf` file to `/etc/init/`, and you'll be good to go.
